Tests/test_Pathway.py

```diff
@@ -120,14 +125,21 @@ def test_Nodes(self):
     def test_Edges(self):
         a = MultiGraph(["a", "b", "c", "d"])
         a.add_edge("a", "b", "label1")
-        self.assertEqual(a.child_edges("a"), [("b", "label1")])  # , "incorrect child edges")
+        self.assertEqual(
+            a.child_edges("a"), [("b", "label1")]
+        )  # , "incorrect child edges")
```

Comment:
Again, the commented out error message can be removed (here and the three which follow).

Reply:
Done, thanks!